chapter8/Example_8_2_Recursive_Circles_Twice/Example_8_2_Recursive_Circles_Twice.pde

```diff
@@ -0,0 +1,27 @@
+// The Nature of Code
+// Daniel Shiffman
+// http://natureofcode.com
+
+// Simple Recursion
+
+void setup() {
+  size(640, 360);
+}
+
+void draw() {
+  background(255);
+  drawCircles(width / 2, height / 2, 320);
+  noLoop();
+}
+
+void drawCircles(float x, int y, float radius) {
+  stroke(0);
+  strokeWeight(2);
+  noFill();
+  circle(x, y, radius * 2);
+  if (radius > 4) {
+    //{!2} drawCircles() calls itself twice. For every circle, a smaller circle is drawn to the left and the right.
```

Comment:
```suggestion
    // drawCircles() calls itself twice. For every circle, a smaller circle is drawn to the left and the right.
```

Just an FYI that extraneous formatting for the book markup can be removed (I know it's in the p5.js examples, I'm slowly trying to get rid of it!)